coordinator/src/services/translation/request/dispense/dispense-notification.ts

```diff
@@ -277,16 +295,10 @@ export function getOrganisationPerformer(fhirFirstMedicationDispense: fhir.Medic
   return fhirFirstMedicationDispense.performer.find(p => p.actor.type === "Organization")
 }
 
-export function getFhirGroupIdentifierExtension(
+export function getMedicationDispenseContained(
   fhirFirstMedicationDispense: fhir.MedicationDispense
-): fhir.ExtensionExtension<fhir.Extension> {
-  const fhirAuthorizingPrescriptionExtensions =
-    fhirFirstMedicationDispense.authorizingPrescription.flatMap(e => e.extension)
-  const fhirGroupIdentifierExtension = getExtensionForUrl(
-    fhirAuthorizingPrescriptionExtensions,
-    "https://fhir.nhs.uk/StructureDefinition/Extension-DM-GroupIdentifier",
-    "MedicationDispense.authorizingPrescription") as fhir.ExtensionExtension<fhir.Extension>
-  return fhirGroupIdentifierExtension
+): fhir.MedicationRequest | undefined {
+  return fhirFirstMedicationDispense.contained.find(p => p.resourceType === "MedicationRequest")
```

Comment:
i think if we were to do this properly, we would save the value of `MedicationDispense.authorizingPrescription` and check contained to see if any resources have the same `id` value as the `reference` in `authorizingPrescription`

Reply:
done